### DISCUSSION TITLE
Where can I set concurrentNodeIterators

### DISCUSSION THREAD

**Discussion by arkansas-coder** [requarks/wiki#6276] [Help / Questions]
Please ignore, I see that it has been addressed previously, and I don't know how to delete it.

Raspberry Pi OS
Docker 
Wiki.js 2.5.297
Postgres 15.1

I am getting the error `Error when running job render-page: This NodeIterator is no longer working. More than undefined iterators are being used concurrently. You can increase the 'concurrentNodeIterators' option to make this error go away.`, but I cannot find where to change the option. How can I change its value?

**Comment by rafapeiter** [comment]
Same problem here. Did you solve it?
